libraries/ui-library-angular/src/lib/stencil-generated/components.ts

```diff
@@ -1197,15 +1197,15 @@ export declare interface SixSelect extends Components.SixSelect {
 
 
 @ProxyCmp({
-  inputs: ['open', 'position', 'toggled', 'width'],
+  inputs: ['open', 'position', 'toggled'],
   methods: ['toggle', 'show', 'hide', 'selectItemByIndex', 'selectItemByName']
 })
 @Component({
   selector: 'six-sidebar',
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: '<ng-content></ng-content>',
   // eslint-disable-next-line @angular-eslint/no-inputs-metadata-property
-  inputs: ['open', 'position', 'toggled', 'width'],
```

Comment:
This would mean we have a breaking change, I guess we could include this in the release which also targets node 18 and above but not before.

cc: @ibirrer